deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -389,6 +429,13 @@ cephFileSystems:
       metadataServer:
         activeCount: 1
         activeStandby: true
+        resources:
+          limits:
+            cpu: "500m"
+            memory: "4096Mi"
+          requests:
+            cpu: "500m"
```

MDS is going to need more cpu depending on load. How about requests of 1cpu and limits of 2cpu?
```suggestion
            cpu: "1000m"
```